tracing-core/src/dispatcher.rs

```diff
@@ -800,4 +807,10 @@ mod test {
 
         with_default(&Dispatch::new(TestSubscriber), || mk_span())
     }
+
+    #[test]
+    fn create_default_dispatcher() {
+        let default_dispatcher = Dispatch::default();
+        assert!(default_dispatcher.is::<NoSubscriber>());
```

Comment:
After this, I think it would be good to try setting a default dispatcher, and then asserting that `Dispatch::default` returns that dispatcher. So, we can add something like:
```rust
    let _guard = set_default(&Dispatch::new(TestSubscriber));
    let default_dispatcher = Dispatch::default();
    assert!(default_dispatcher.is::<TestSubscriber>());
```
We'd also need to either add a no-op `TestSubscriber` type in this test as well, or move one out of another test function into the test module scope.

Reply:
Ah, gotcha. Will try to add that test.

Also, sorry for asking this, but can you give me detail regarding the no-op `TestSubscriber`? What's the specific test for that?

Reply:
actually, scrub that, I think I know what you mean, will add that test in the PR.